<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 100pt &quot;Blackadder ITC&quot;;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_userName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>USERNAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>PASSWORD:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_userName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_Password">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_commit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>commit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
